### ISSUE TITLE
Blog Post: Image Scanning and Code Quality for Dockerfiles

### ISSUE THREAD

**Issue by mcastellin** [StatusNeo/Hacktoberfest_2020#13]
Hi! My name is Manuel and I would like to contribute to your repository with a blog post on Docker container security.

I have been a Docker user for a long time now and the next thing I want to learn about is Container Security. With this post I want to take the opportunity to scout the internet for all the different ways to perform container security scans and ensure code quality in Dockerfiles and organise the lessons learned in a post.

Here is a list of things I want to look for:
- container security scanner Aqua Microscanner
- the new addition to the Docker CLI `docker scan` with Snyk integration
- the best option available for Dockerfile quality assurance, Hadolint or other? 

I would love to work on this post here! Thanks for the opportunity!

**Comment by NishkarshRaj** [comment]
Congrats! You created your first PR on this repository :+1: We will revert shortly' first issue

**Comment by NishkarshRaj** [comment]
@mcastellin don't mind my bot stating an issue as PR! The GitHub Actions has been modified 😄 

I have created a blog topic and assigned it you here https://github.com/StatusNeo/Hacktoberfest_2020/blob/main/blogs/Readme.md

Please create a PR and mention it in this issue.

**Comment by mcastellin** [comment]
Yeah I though it could not have been you replying in 3 seconds 😅 thanks man!

**Comment by NishkarshRaj** [comment]
> 
> 
> Yeah I though it could not have been you replying in 3 seconds 😅 thanks man!

That's the magic of GitHub Actions - also made using docker 😉
